InvSee/Commands.cs

```diff
@@ -24,8 +24,10 @@ public static void DoInvSee(CommandArgs args)
 			{
 				bool restored = info.Restore(args.Player);
 
-				if (restored)
-					args.Player.PluginSuccessMessage("Restored your inventory.");
+				if (args.Player.Dead)
```

Comment:
If you're looking into disabling InvSee while the user is dead, then why perform this check after you've already run the `Restore` method? (two lines above). This check needs to go before that.